api/model.go

```diff
@@ -1951,6 +1951,15 @@ func toBackupResource(b *longhorn.Backup) *Backup {
 		return nil
 	}
 
+	backupTargetName := b.Status.BackupTargetName
+	if backupTargetName == "" {
+		backupTargetName = b.Labels[types.LonghornLabelBackupTarget]
+	}
+	volumeName := b.Status.VolumeName
+	if volumeName == "" {
+		volumeName = b.Labels[types.LonghornLabelBackupVolume]
+	}
```

Comment:
Why can `b.Status.BackupTargetName` and `b.Status.VolumeName` be empty?

Reply:
Discussed with @mantissahz. A failed backup might have empty values because the information cannot be retrieved from backend.